api.md

```diff
@@ -171,7 +172,115 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_e
 ```
 
 ---
-## attach_tablespace() [](attach_tablespace)
+### add_data_node() [](add_data_node)
+
+Create a new data node for use when creating distributed hypertables.
+
+To create a distributed hypertable it is necessary to define the data
+nodes where the data is going to be stored, which this utility is used
+for.
+
+The utility will attempt to bootstrap the data node by creating the
+database where the remote hypertables will later be created and also
+load the TimescaleDB extension on the data node.
+
+#### Errors
+
+* An error will be given if an attempt is made to run the function
+  inside a transaction.
+* An error will be given if the database already exists on the data
+  node and `if_not_exists` is not set.
+
+#### Required Arguments [](add_data_node-required-arguments)
+
+| Name        | Description       |
+| ----------- | -----------       |
+| `node_name` | Name of the node. |
+
+##### Parameter Description
+
+* `node_name` - Name to use for the node locally.
+
+#### Optional Arguments [](add_data_node-optional-arguments)
+
+| Name                 | Description                                           |
+|----------------------|-------------------------------------------------------|
+| `host`               | Host name for the remote data node.                   |
```

Comment:
Should we put the default setting in this table instead of below? Or, is this the established pattern?

Reply:
The tables become exceedingly wide and the default is occasionally derived from other parameters, hence require more elaborate descriptions, which is why I went with this approach.